src/main/java/logic/Dealer.java

```diff
@@ -0,0 +1,32 @@
+//package logic;
```

Comment:
Czemu ten plik jest cały zakomentowany, a w innym package mamy też klasę Dealer? W sensie to musi tu zostać z jakiegoś powodu czy możnaby to usunąć?

Reply:
Fakt to można usunąć. Stara implemetacje klasa, jeszcze jak były stare nazwy pakietów